package-lock.json

```diff
@@ -225,9 +225,9 @@
           }
         },
         "@babel/parser": {
-          "version": "7.7.4",
-          "resolved": "https://registry.npmjs.org/@babel/parser/-/parser-7.7.4.tgz",
-          "integrity": "sha512-jIwvLO0zCL+O/LmEJQjWA75MQTWwx3c3u2JOTDK5D3/9egrWRRA0/0hk9XXywYnXZVVpzrBYeIQTmhwUaePI9g==",
+          "version": "7.7.5",
```

Comment:
For my own education: why do we change this?

Reply:
@emma-green frontend-build was updated after I pulled from master, so it needs to be updated to the latest release.